hardware/national_instruments_x_series.py

```diff
@@ -1562,6 +1572,19 @@ def set_up_odmr(self, counter_channel=None, photon_source=None,
             # otherwise, it will be low until task starts, and MW will receive wrong pulses.
             daq.DAQmxStopTask(self._scanner_clock_daq_task)
 
+            if self._lock_in_active:
+                ptask = daq.TaskHandle()
+                daq.DAQmxCreateTask('ODMRPulser', daq.byref(ptask))
+                daq.DAQmxCreateDOChan(ptask, self._pulse_out_channel, "ODMRPulserChannel", daq.DAQmx_Val_ChanForAllLines)
```

Comment:
This can be written in a way that does not use/overwrite all 32 channels of Port0.
Only the 2 lines that are needed should be reserved and written.

Reply:
still needs to be tested